### ISSUE TITLE
`TypeError: entity-schema claim parser isn't implemented`

### ISSUE THREAD

**Issue by egonw** [maxlath/wikibase-sdk#133]
- wikibase-sdk version: 7.15.0 (yeah, I know; I'll see if I can update citation.js asap)
- Environment: v18.19.0 (cli)

I am using citation.js to create formatted references where the reference is a Wikidata item. I get this error, which likely has to do with the added support for the entity-schema data type:

```
/home/egonw/var/Projects/hub/cdkbook-source/node_modules/wikibase-sdk/lib/helpers/parse_claim.js:109
      return parsers[datatype](datavalue, options)
                              ^

TypeError: entity-schema claim parser isn't implemented
        Claim id: Q235557$3f27abc2-4b7f-085c-cd06-520c19bf0f59
        Please report to https://github.com/maxlath/wikibase-sdk/issues
    at parse (/home/egonw/var/Projects/hub/cdkbook-source/node_modules/wikibase-sdk/lib/helpers/parse_claim.js:109:31)
    at simplifyClaim (/home/egonw/var/Projects/hub/cdkbook-source/node_modules/wikibase-sdk/lib/helpers/simplify_claims.js:79:13)
    at /home/egonw/var/Projects/hub/cdkbook-source/node_modules/wikibase-sdk/lib/helpers/simplify_claims.js:37:19
    at Array.map (<anonymous>)
    at simplifyPropertyClaims (/home/egonw/var/Projects/hub/cdkbook-source/node_modules/wikibase-sdk/lib/helpers/simplify_claims.js:37:6)
    at Object.simplifyClaims (/home/egonw/var/Projects/hub/cdkbook-source/node_modules/wikibase-sdk/lib/helpers/simplify_claims.js:15:24)
    at simplifyIfDefined (/home/egonw/var/Projects/hub/cdkbook-source/node_modules/wikibase-sdk/lib/helpers/simplify_entity.js:39:48)
    at simplifyEntity (/home/egonw/var/Projects/hub/cdkbook-source/node_modules/wikibase-sdk/lib/helpers/simplify_entity.js:17:5)
    at /home/egonw/var/Projects/hub/cdkbook-source/node_modules/wikibase-sdk/lib/helpers/simplify_entity.js:49:16
    at Array.reduce (<anonymous>)

Node.js v18.19.0
```

**Comment by egonw** [comment]
With 10.0.2:

```
/home/egonw/var/Projects/hub/cdkbook-source/node_modules/@citation-js/plugin-wikidata/node_modules/wikibase-sdk/lib/helpers/parse_claim.js:109
      return parsers[datatype](datavalue, options)
                              ^

TypeError: entity-schema claim parser isn't implemented
        Claim id: Q235557$3f27abc2-4b7f-085c-cd06-520c19bf0f59
        Please report to https://github.com/maxlath/wikibase-sdk/issues
    at parse (/home/egonw/var/Projects/hub/cdkbook-source/node_modules/@citation-js/plugin-wikidata/node_modules/wikibase-sdk/lib/helpers/parse_claim.js:109:31)
    at simplifyClaim (/home/egonw/var/Projects/hub/cdkbook-source/node_modules/@citation-js/plugin-wikidata/node_modules/wikibase-sdk/lib/helpers/simplify_claims.js:79:13)
    at /home/egonw/var/Projects/hub/cdkbook-source/node_modules/@citation-js/plugin-wikidata/node_modules/wikibase-sdk/lib/helpers/simplify_claims.js:37:19
    at Array.map (<anonymous>)
    at simplifyPropertyClaims (/home/egonw/var/Projects/hub/cdkbook-source/node_modules/@citation-js/plugin-wikidata/node_modules/wikibase-sdk/lib/helpers/simplify_claims.js:37:6)
    at Object.simplifyClaims (/home/egonw/var/Projects/hub/cdkbook-source/node_modules/@citation-js/plugin-wikidata/node_modules/wikibase-sdk/lib/helpers/simplify_claims.js:15:24)
    at simplifyIfDefined (/home/egonw/var/Projects/hub/cdkbook-source/node_modules/@citation-js/plugin-wikidata/node_modules/wikibase-sdk/lib/helpers/simplify_entity.js:39:48)
    at simplifyEntity (/home/egonw/var/Projects/hub/cdkbook-source/node_modules/@citation-js/plugin-wikidata/node_modules/wikibase-sdk/lib/helpers/simplify_entity.js:17:5)
    at /home/egonw/var/Projects/hub/cdkbook-source/node_modules/@citation-js/plugin-wikidata/node_modules/wikibase-sdk/lib/helpers/simplify_entity.js:49:16
    at Array.reduce (<anonymous>)

Node.js v18.19.0
```

**Comment by maxlath** [comment]
I added `entity-schema` support in 279848a, published in `v10.1.0`. Thanks Egon for reporting!
